### PR TITLE
switch to GitHub actions

### LINKED ISSUE #158: transition from Travis CI to GitHub actions
Use the free GitHub actions.

**Important** because we ran out of Travis credits.

### PR DESCRIPTION
**Pull request by orbeckst**
- close #158 
- use GitHub actions (start with minimal tests)
- make tests pass

### CODE REVIEW

File: mdpow/tests/test_solvation.py [hunk 1]
```diff
@@ -52,7 +52,12 @@ def test_solvation_octanol(setup, ff):
 def test_solvation_cyclohexane(setup):
     solvation(setup, "cyclohexane")
 
-@pytest.mark.xfail("gromacs.release().startswith('2019')")
+@pytest.mark.xfail(gromacs.release.startswith('4')
+                   or gromacs.release.startswith('5')
+                   or gromacs.release.startswith('2016')
+                   or gromacs.release.startswith('2018'),
```
Comment (VOD555) [inline mdpow/tests/test_solvation.py:58]: 2018 may work with mixed solvent.

Comment (orbeckst) [reply mdpow/tests/test_solvation.py]: Let's see if it works.

Comment (orbeckst) [reply mdpow/tests/test_solvation.py]: You were right, 2018 also passes the test.

**Review by VOD555 — APPROVED** [review]

**Comment by orbeckst** [comment]
@VOD555 @iorga do is it sufficient to test for GROMACS 4.6.5, 2018, 2019, 2021 ? Or do we need to add 2020 (the only other one available on bioconda)? Or can I drop, say 2019?

**Comment by orbeckst** [comment]
Comments still welcome, but will address anything in separate PRs.

**Comment by VOD555** [comment]
I think we can drop 2019 and only keep 4.6.3, 2018 and 2021.

**Comment by iorga** [comment]
Currently, I am using only 2020.3, so it would be good to add 2020 to the list. I'm OK with dropping 2019, I have never used it.

**Pull request merged**